blockchains/erc20/lib/contract_overwrite.js

```diff
@@ -23,87 +19,50 @@ class ContractOverwrite {
   }
 }
 
-class ContractEditor {
```

Comment:
Removing this class and using its methods as functions. I think the code is subjectively easier to understand now.

Reply:
I thought about this myself a few days ago. nice